src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -761,6 +760,13 @@ public override void DeleteGeometryForIdentifier(string identifier, bool request
             DeleteGeometries(geometryModels, requestUpdate); 
         }
 
+        protected override void OnRenderPackagesUpdated(NodeModel node, IEnumerable<IRenderPackage> packages)
```

Comment:
@mjkkirschner you're right. Not removing the render packages always will land us back in the same situation that https://github.com/DynamoDS/Dynamo/pull/6157 was trying to address in the first place. In that case you could simply change the base class method to look like this: 
```
protected virtual void OnRenderPackagesUpdated(NodeModel node, IEnumerable<IRenderPackage> packages)
        {
            
            RemoveGeometryForNode(node);
            if (packages.Any())
            {
                AddGeometryForRenderPackages(packages);
            }
        }
```